### PR TITLE
[backplane-2.3] orphan the agent install namespace

### PR DESCRIPTION
**Pull request by zhujian7**
issue reference: 
- https://issues.redhat.com/browse/ACM-6864?focusedId=22811617&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22811617
- https://issues.redhat.com/browse/ACM-7114

### CODE REVIEW

**Comment by zhujian7** [comment]
cc @tesshuflower

File: pkg/addon/manager/manifests/templates/namespace.yaml [hunk 1]
```diff
@@ -2,3 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .AddonInstallNamespace }}
+{{ if eq .AddonInstallNamespace "open-cluster-management-agent-addon" }}
```
Comment (qiujian16) [inline pkg/addon/manager/manifests/templates/namespace.yaml:5]: is this eq or ne?

Comment (zhujian7) [reply pkg/addon/manager/manifests/templates/namespace.yaml:5]: eq, if it is open-cluster-management-agent-addon, mark it as "deletion-orphan"

**Comment by zhujian7** [comment]
ACM 2.8.2 is open for PRs
/unhold

**Pull request merged**